ec/resources/waf/managed_rule.go

```diff
@@ -283,11 +308,57 @@ func ResourceManagedRuleCreate(ctx context.Context, d *schema.ResourceData, m in
 
 	d.SetId(resp.ID)
 
-	return diags
+	return ResourceManagedRuleRead(ctx, d, m)
 }
 
 func ResourceManagedRuleRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
+
+	// Retrieve data for Read call
+	accountNumber := d.Get("account_number").(string)
+	ruleID := d.Id()
+
+	log.Printf("[INFO] Reading WAF Managed Rule ID %s for Account >> %s", ruleID, accountNumber)
+
+	// Initialize WAF Service
+	config := m.(**api.ClientConfig)
+	wafService, err := buildWAFService(**config)
+
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	// Retrieve Managed Rule
+	resp, err := wafService.GetManagedRule(accountNumber, ruleID)
+	if err != nil {
+		d.SetId("")
+		return diag.FromErr(err)
+	}
+	log.Printf("[INFO] Retrieved Managed Rule: %+v", resp)
```

Comment:
Just a note for future reference: printing a struct using the `%v` format doesn't work the best if the struct has pointer properties. It looks like ManagedRule does not have any pointer properties so it works out fine here.

If the struct has pointer properties then we have to print those properties manually